check-vulnerabilities.ps1

```diff
@@ -1,33 +1,60 @@
-$projectPath = ".\src\Azure.Functions.Cli"
-$projectFileName = ".\Azure.Functions.Cli.csproj"
-$logFilePath = "..\..\build.log"
-if (-not (Test-Path $projectPath))
-{
-    throw "Project path '$projectPath' does not exist."
+param
+(
+    [String[]]
+    $CsprojFilePath
+)
+
+if (-not $CsprojFilePath) {
+    $CsprojFilePath = @(
+        "$PSScriptRoot/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"
+        "$PSScriptRoot/test/Azure.Functions.Cli.Tests/Azure.Functions.Cli.Tests.csproj"
+        "$PSScriptRoot/build/Build.csproj"
+    )
 }
 
-cd $projectPath
+$logFilePath = "$PSScriptRoot/build.log"
+foreach ($projectFilePath in $CsprojFilePath) {
```

Comment:
Thinking aloud, I wonder if we would want to parallelize these upstream of the call into the script, I suppose that's supported by the parameter being included here. But we should probably ensure that gets tracked, as it will be much more efficient.